### ISSUE TITLE
NoClassDefFoundError on Firebase-Perf

### ISSUE THREAD

**Issue by ppamorim** [firebase/firebase-android-sdk#126]
Environment:

  * Android Studio version: 3.1.4
  * Firebase Component: Performance
  * Component version: 16.1.2

Problem:

#### Steps to reproduce:

What happened? How can we make the problem occur?

I got this crash report on Crashlytics. The device is a Samsung Galaxy A3, Android version 6.0.1.

#### Relevant Code:

```
Fatal Exception: java.lang.NoClassDefFoundError: com.google.android.gms.d.f.gm
       at com.google.android.gms.internal.firebase-perf.zzc.zzc(Unknown Source:54)
       at com.google.firebase.perf.network.FirebasePerfOkHttpClient.zza(Unknown Source:49)
       at com.google.firebase.perf.network.zzg.onResponse(Unknown Source:22)
       at okhttp3.RealCall$AsyncCall.execute(RealCall.java:153)
       at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
       at java.lang.Thread.run(Thread.java:818)
```

The obfuscation code is uploaded but it is not mapping correctly. I've followed this tutorial here but nothing has changed: https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports

**Comment by tejasd** [comment]
Hi @ppamorim - Thanks for the report!

This is a known issue in version 16.1.2 of the firebase performance sdk. It has been challenging to reproduce, but we have a potential fix in version 16.2.0. Would you be able to upgrade to 16.2.0 to see if this still persists?

Additionally, I would also recommend upgrading to Android Studio version 3.2.1 as we've noticed issues with code minifying when using version 3.1.x.

As for the obfuscated code - because we minify and obfuscate the library before we ship it out, those proguard maps are not available to third party developers right now, so it's working as intended!

**Comment by ppamorim** [comment]
@tejasd Thank you, I will do it.
